pkgs/development/compilers/llvm/git/llvm/default.nix

```diff
@@ -1,423 +0,0 @@
-{ lib, stdenv, llvm_meta
-, pkgsBuildBuild
-, monorepoSrc
-, runCommand
-, fetchpatch
-, cmake
-, darwin
-, ninja
-, python3
-, libffi
-, enableGoldPlugin ? (!stdenv.isDarwin && !stdenv.targetPlatform.isWasi)
```

Comment:
This is `enableGoldPlugin ? libbfd.hasPluginAPI` in 15 because https://github.com/nixos/nixpkgs/commit/b9a9dfc3d4596825d98bb87b7f488ff1a5d765f1 wasn't ported to git